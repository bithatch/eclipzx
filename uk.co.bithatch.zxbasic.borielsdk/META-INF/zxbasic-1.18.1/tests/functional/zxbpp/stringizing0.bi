

#define z(x) # x

z(1)
