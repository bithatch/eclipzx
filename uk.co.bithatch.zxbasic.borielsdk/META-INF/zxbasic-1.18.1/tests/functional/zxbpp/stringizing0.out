#line 1 "stringizing0.bi"


#line 4 "stringizing0.bi"

"1"
